logging/src/main/java/se/fortnox/reactivewizard/logging/LoggingFactory.java

```diff
@@ -54,6 +60,22 @@ public void init() {
             props.setProperty("log4j.logger." + e.getKey(), e.getValue());
         }
         LoggingConfiguration.getInstance().configure(props);
+
+        /*
+        Provides a facility to reconfigure JUL via a reactivewizard config.
+         */
+        if(julLoggingActivation) {
+	        configureJUL();
+        }
+    }
+
+    private void configureJUL() {
+        final InputStream stream = LoggingFactory.class.getClassLoader().getResourceAsStream("rw-jul-logging.properties");
```

Comment:
Is rw-jul-logging.properties a file that should have been in the PR?

Reply:
And if the file is not present the startup will fail I guess. So perhaps include this getResourceAsStream row in the try catch statement

Reply:
Regarding rw-jul-logging.properties, see a pull request hidden from the public eyes :D

Reply:
...but shouldn't there be a default config that does not override the formatter?

Reply:
JUL has a default configuration under <path to JRE>/lib/logging.properties which I have no idea when it activates :)